src/App.tsx

```diff
@@ -60,7 +60,10 @@ const styles = StyleSheet.create({
 });
 
 function RenderDrawerItem(props, i: number) {
-  const isDrawerOpen = getIsDrawerOpenFromState(props.navigation.getState());
+  const isDrawerOpen =
+    getDrawerStatusFromState(props.navigation.getState()) == 'open'
+      ? true
+      : false;
```

Comment:
```suggestion
    getDrawerStatusFromState(props.navigation.getState()) == 'open';
```
Isn't the `? true : false` here redundant?